src/metabase/public_settings.clj

```diff
@@ -147,7 +147,7 @@
   :default 60)
 
 (defsetting query-caching-ttl-ratio
-  (str (deferred-tru "To determine how long each saved question''s cached result should stick around, we take the query''s average execution time and multiply that by whatever you input here.")
+  (str (deferred-tru "To determine how long each saved question''s cached result should stick around, we take the query''s average execution time and multiply that by whatever you input here. ")
```

Comment:
Is the new space at the end of this string necessary?

Reply:
Yes, you can see here that the sentences currently run together without it:

![image](https://user-images.githubusercontent.com/2223916/71846575-c60de280-307f-11ea-83c2-d2377edec92d.png)

Reply:
Got it. It feels weird to have to add an extra space in a string to get the presentation right and I'd imagine some editors might even remove it on save so at the very least we'd want a code comment but my guess is there's some other way to fix that that would be better and not so fragile.

Reply:
Yeah, the other way to do this is to just combine the two sentences into a single `deferred-tru`, but I wasn't sure if I was knocking down a clojure Chesterton Fence in doing that.

Reply:
I think this might work if we _always_ want the space:

before: `here. ")`
after: `here.") " "`

However, I think it'd be best to leave that up to the translator. In some languages, the punctuation can have built-in spacing. e.g. Here's what you get when you type "." on a Chinese keyboard: "。". I might be missing something, but could these sentences be combined?

Reply:
Yeah if these are meant to be displayed together it'd be nice to just have them together in one string and also have that context for translators.